baseplate/lib/edge_context.py

```diff
@@ -81,9 +81,21 @@ def oauth_client_id(self) -> Optional[str]:
     def oauth_client_type(self) -> Optional[str]:
         raise NotImplementedError
 
+    @property
+    def scopes(self) -> Set[str]:
+        raise NotImplementedError
+
+    @property
+    def loid_id(self) -> Optional[str]:
```

Comment:
why the extra id? doesn't loid mean logged out id?

Reply:
the loid naming is very confusing sometimes means id sometimes means dict of loid and create_ms. i am open to any suggestino

Reply:
how about just `loid`